### PR TITLE
Formal and light content improvements of mock docs

### PR DESCRIPTION
**Pull request by andy-maier**
See commit message.

### CODE REVIEW

File: pywbem_mock/_baseprovider.py [hunk 1]
```diff
@@ -290,50 +291,54 @@ def remove_namespace(self, namespace):
...
-        Returns:
+        Only the valid Interop namespace names can be used to register Interop
+        namespace providers.
+
+        This list is defined in :attr:`pywbem.WBEMServer.INTEROP_NAMESPACES`.
 
-          List of :term:`string`: containing the valid names for
-          namespaces that can be interop namespaces. Any name not on this list
-          cannot be an interop provider
+        Note that namespace names need to be considered case insensitive.
```
Comment (KSchopmeyer) [inline pywbem_mock/_baseprovider.py:301]: Remove "Note that"

File: pywbem_mock/_baseprovider.py [hunk 2]
```diff
@@ -452,11 +457,11 @@ class are returned. `None` means not supplied by client and the
           filtered in accord with the local_only, etc. arguments.
 
         Raises:
-          :class:`~pywbem.CIMError`: (CIM_ERR_NOT_FOUND) if class not found in
-          CIM repository.
 
+          :class:`~pywbem.CIMError`: (CIM_ERR_NOT_FOUND) if class not found in
```
Comment (KSchopmeyer) [inline pywbem_mock/_baseprovider.py:461]: We are inconsistent on whether the status_code_name is in paren. Most other places it is not. Actually I like the paren.

Comment (andy-maier) [reply pywbem_mock/_baseprovider.py:461]: DONE - Added parenthesis all over the uses in pywbem_mock.

File: pywbem_mock/_baseprovider.py (hunk 1 above)
Comment (andy-maier) [reply pywbem_mock/_baseprovider.py:301]: DONE

**Pull request merged**